src/tabs/interfaces.ts

```diff
@@ -78,6 +78,11 @@ export interface TabsProps extends BaseComponentProps {
    * If the tab content is too short, it will stretch. If the tab content is too long, a vertical scrollbar will be shown.
    */
   fitHeight?: boolean;
+  /**
+   * Determines whether the content for a specific tab should be rendered. This can be used to
+   * retain non-active tabs in memory. By default, only the active tab content is rendered.
+   */
+  renderTabContent?: (tab: TabsProps.Tab) => boolean | undefined;
```

Comment:
Should we call this something like `shouldRenderTabContent`? Otherwise it can be confused with a renderer function that determines the actual content.

Reply:
good point, I think it _was_ that in my initial revision, not sure why I changed it: will update!